tests/functional/api/groups/members_test.py

```diff
@@ -21,17 +22,18 @@ def test_it_returns_list_of_members_if_user_has_access_to_private_group(
         self, app, factories, db_session, group, user_with_token, token_auth_header
     ):
         user, _ = user_with_token
-        group.members.append(user)
+        group.members.extend([user, factories.User()])
```

Comment:
The test is `test_it_returns_list_of_members_if_user_has_access_to_private_group`. It's testing that, given that `user` is a member of `group`, the list-group-members API when called by `user` returns a list of the group's members.

The test was previously relying on `factories.Group()` having generated a user for the group's creator and added the creator as a member of the group, and it asserted that the API returned both `user` and `group.creator`.

But the test really has nothing to do with group creators, so I've fixed it to add two members to the group (`user` and another) and to assert that the API returns these two members.